src/gpu/generic/sycl/ref_convolution.cpp

```diff
@@ -134,6 +136,8 @@ status_t ref_convolution_bwd_data_t::init(impl::engine_t *engine) {
 }
 
 status_t ref_convolution_bwd_data_t::execute(const exec_ctx_t &ctx) const {
+    if (memory_desc_wrapper(pd()->dst_md()).size() == 0) return status::success;
```

Comment:
```suggestion
    if (memory_desc_wrapper(pd()->diff_src_md()).size() == 0) return status::success;
```

Reply:
Thanks, I am guessing this is the fix for the failing convolution tests? :)

Reply:
yes, that is correct.